azure/azure-release-template.yml

```diff
@@ -44,15 +44,19 @@ extends:
           releaseType: stable
         condition: and(succeeded(), eq(variables['run_smoke_tests'], 'true'))
       - bash: |
+          #!/bin/bash
+
+          # shellcheck disable=SC2155
```

Comment:
Could we not just do what it's telling us to and split the declaration from the assignment?

Reply:
We could. I googled the error and it said it was a false positive a lot of the time, but i suppose we may aswell.